qutip/wigner.py

```diff
@@ -603,20 +633,86 @@ def qfunc(state, xvec, yvec, g=sqrt(2)):
 # |psi>   = the state in fock basis
 # |alpha> = the coherent state with amplitude alpha
 #
-def _qfunc_pure(psi, alpha_mat):
+def _qfunc_pure(psi, alpha_mat, precompute=False):
```

Comment:
`None` is the correct type to indicate absence of value, rather than `False` - it's a Python guarantee that all references to `None` are referentially equal.